src-java/kilda-persistence-tinkerpop/src/main/java/org/openkilda/persistence/ferma/frames/FlowFrame.java

```diff
@@ -197,12 +198,20 @@
     public abstract void setProtectedReversePathId(PathId protectedReversePathId);
 
     @Override
-    @Property(GROUP_ID_PROPERTY)
-    public abstract String getGroupId();
+    @Property(DIVERSE_GROUP_ID_PROPERTY)
+    public abstract String getDiverseGroupId();
```

Comment:
Will this work with the blue-green deployment approach? DB migration doesn't guarantee that  when it's done the previous version of the system wont update old fields in DB.

Reply:
This approach will not work with the blue-green deployment. The approach has been reworked. Now GROUP_ID_PROPERTY is used, and we write diverse group id value to GROUP_ID_PROPERTY and DIVERSE_GROUP_ID_PROPERTY at the same time. Second step is starting use DIVERSE_GROUP_ID_PROPERTY (#4404). Third step is removing group id field (#4405).